aspnetcore/fundamentals/minimal-apis/bindStreamPipeReader/7.0-samples/PipeStreamToBackgroundQueue/Program.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Channels;
+using BackgroundQueueService;
+var builder = WebApplication.CreateBuilder(args);
+// Create a channel to send data to the background queue.
+builder.Services.AddSingleton<Channel<Stream>>((_) => Channel.CreateBounded<Stream>(100));
```

Comment:
Maybe this should be a `Channel<byte[]>` instead?

Reply:
let me try that out